<ui version="4.0" >
 <class>ConfigurationWidget</class>
 <widget class="QWidget" name="ConfigurationWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="lConfigName" >
       <property name="text" >
        <string>Online reader accounts:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QTreeWidget" name="list_readerList" >
       <property name="rootIsDecorated" >
        <bool>false</bool>
       </property>
       <property name="columnCount" >
        <number>2</number>
       </property>
       <column>
        <property name="text" >
         <string>Type</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Description</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QPushButton" name="b_add" >
         <property name="text" >
          <string>Add...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_update" >
         <property name="text" >
          <string>Modify...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="b_remove" >
         <property name="text" >
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="lDeletePolicy" >
       <property name="text" >
        <string>Delete feeds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_deleteFeeds" />
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>400</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
